<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuestionEditWindow</class>
 <widget class="QMainWindow" name="QuestionEditWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit question</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="question_text_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Question text:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="question_text_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>261</width>
      <height>71</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Insert the text of the question here...</string>
    </property>
   </widget>
   <widget class="QLabel" name="aswer_signature_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Answer signature:</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="answer_signature_widget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>351</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="free_value">
     <attribute name="title">
      <string>Free value</string>
     </attribute>
     <widget class="QTabWidget" name="value_answer_signature_widget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>271</width>
        <height>151</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="int">
       <attribute name="title">
        <string>Integer number</string>
       </attribute>
       <widget class="QLabel" name="min_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Minimum value:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="min_spin_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>52</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
       <widget class="QLabel" name="max_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum value:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="max_spin_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>52</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
       <widget class="QLabel" name="default_label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>91</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Default value:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="default_spin_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>52</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <number>9999999</number>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="double">
       <attribute name="title">
        <string>Floating point number</string>
       </attribute>
       <widget class="QLabel" name="default_label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>91</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Default value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="max_label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Maximum value:</string>
        </property>
       </widget>
       <widget class="QLabel" name="min_label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>111</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Minimum value:</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="min_double_spin_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>0</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="max_double_spin_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="default_double_spin_box">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>71</width>
          <height>32</height>
         </rect>
        </property>
        <property name="maximum">
         <double>9999999.990000000223517</double>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="signle_option">
     <attribute name="title">
      <string>Single option</string>
     </attribute>
     <widget class="QLabel" name="option_slabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Options:</string>
      </property>
     </widget>
     <widget class="QScrollArea" name="options_scroll_area">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>401</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>327</width>
         <height>397</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="multiple_options">
     <attribute name="title">
      <string>Multiple options</string>
     </attribute>
     <widget class="QScrollArea" name="options_scroll_area_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>331</width>
        <height>401</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>327</width>
         <height>397</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="option_slabel_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>58</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Options:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
